test/integration/targets/apt_virtual_pkg_fix/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Create directory for test .deb
+  ansible.builtin.file:
+    path: /tmp/virtual-test/DEBIAN
+    state: directory
+
+- name: Create a test .deb file that depends on a virtual package
+  ansible.builtin.shell: |
+    cat > /tmp/virtual-test/DEBIAN/control << EOF
+    Package: virtual-test-pkg
+    Version: 1.0
+    Architecture: all
+    Description: Test package depending on virtual package
+    Depends: mail-transport-agent
+    EOF
+    dpkg-deb --build /tmp/virtual-test /tmp/virtual-test.deb
+  args:
+    creates: /tmp/virtual-test.deb
```

Comment:
We download pre-built deb packages from `https://ci-files.testing.ansible.com/test/integration/targets/apt/`. For now, create a temporary location from which CI can download the `.deb` file. Later, I will make a `.deb` file and upload it to `https://ci-files.testing.ansible.com`.